src/wp-includes/rest-api/endpoints/class-wp-rest-font-families-controller.php

```diff
@@ -0,0 +1,554 @@
+					'type'        => 'array',
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'items'       => array(
+						'type' => 'integer',
+					),
+				),
+				// Font family settings come directly from theme.json schema
+				// See https://schemas.wp.org/trunk/theme.json
+				'font_family_settings' => array(
+					'description'          => __( 'font-face definition in theme.json format.' ),
+					'type'                 => 'object',
+					'context'              => array( 'view', 'edit', 'embed' ),
+					'properties'           => array(
+						'name'       => array(
+							'description' => __( 'Name of the font family preset, translatable.' ),
+							'type'        => 'string',
+							'arg_options' => array(
+								'sanitize_callback' => 'sanitize_text_field',
+							),
+						),
+						'slug'       => array(
+							'description' => __( 'Kebab-case unique identifier for the font family preset.' ),
+							'type'        => 'string',
+							'arg_options' => array(
+								'sanitize_callback' => 'sanitize_title',
+							),
+						),
+						'fontFamily' => array(
+							'description' => __( 'CSS font-family value.' ),
+							'type'        => 'string',
+							'arg_options' => array(
+								'sanitize_callback' => array( 'WP_Font_Utils', 'sanitize_font_family' ),
+							),
+						),
+						'preview'    => array(
+							'description' => __( 'URL to a preview image of the font family.' ),
+							'type'        => 'string',
+							'format'      => 'uri',
+							'default'     => '',
+							'arg_options' => array(
+								'sanitize_callback' => 'sanitize_url',
+							),
+						),
+					),
+					'required'             => array( 'name', 'slug', 'fontFamily' ),
+					'additionalProperties' => false,
+				),
+			),
+		);
+
+		$this->schema = $schema;
+
+		return $this->add_additional_fields_schema( $this->schema );
+	}
+
+	/**
+	 * Retrieves the item's schema for display / public consumption purposes.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @return array Public item schema data.
+	 */
+	public function get_public_item_schema() {
+
+		$schema = parent::get_public_item_schema();
+
+		// Also remove `arg_options' from child font_family_settings properties, since the parent
+		// controller only handles the top level properties.
+		foreach ( $schema['properties']['font_family_settings']['properties'] as &$property ) {
+			unset( $property['arg_options'] );
+		}
+
+		return $schema;
+	}
+
+	/**
+	 * Retrieves the query params for the font family collection.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @return array Collection parameters.
+	 */
+	public function get_collection_params() {
+		$query_params = parent::get_collection_params();
+
+		// Remove unneeded params.
+		unset( $query_params['after'] );
+		unset( $query_params['modified_after'] );
+		unset( $query_params['before'] );
+		unset( $query_params['modified_before'] );
+		unset( $query_params['search'] );
+		unset( $query_params['search_columns'] );
+		unset( $query_params['status'] );
```

Comment:
Nit: you can just unset these in one call:

`unset( $query_params['after'], $query_params['modified_after'], ... )`

Reply:
I'm addressing this in this Gutenberg PR: https://github.com/WordPress/gutenberg/pull/58736. Once merged, I'll commit to this PR.